server/routes/uploadFile.js

```diff
@@ -28,7 +28,7 @@ uploadRouter.route("/upload").post(upload.single("uploadFile"), (req, res) => {
 
   res.send({
     status: 200,
-    filePath: filePath.replace(/\\/g,"/").replace("uploads/", "http://localhost:5001/"),
+    filePath: filePath.replace("uploads/", "http://localhost:5001/"),
```

Comment:
For v2, please make sure to strip out hardcoding of any development urls (http://localhost:5001/) which would cause issues when deployed.